ibis/backends/clickhouse/__init__.py

```diff
@@ -438,9 +438,7 @@ def insert(
         if isinstance(obj, pa.Table):
             return self.con.insert_arrow(name, obj, settings=settings, **kwargs)
```

Comment:
Looks like we don't hit this code path in a test anywhere. I'll add a test in another PR.

Reply:
#10288